_includes/get_a_quote.html

```diff
@@ -8,11 +8,8 @@ <h1>Get a quote</h1>
         We are experts in working in the most efficient way possible, playing to each others' strengths, making sure you get the most value delivered for your money.
     </p>
     <p>
-        If you get in touch, we will first arrange a meeting to talk about your concrete business situation.
-        We will suggest a small project for you to get familiar with how we work, and then hopefully initiate a longer, sustainable relationship, where we help you improve your business's software.
-    </p>
-    <p>
-        Tell us a bit about your business and why you are considering getting help building software. Get in touch if you want to hear from us:
+        Get in touch and we will first arrange a meeting to talk about your concrete business situation.
+        We will suggest a small project for you to get familiar with how we work, and if the fit is right initiate a longer, sustainable relationship, where we help you improve your business's software.
```

Comment:
update to 'we usually suggest a small project'
This makes it sound more flexible